system/jlib/jlog.cpp

```diff
@@ -2645,6 +2647,7 @@ MODULE_EXIT()
     thePassNoneFilter = nullptr;
     thePassLocalFilter = nullptr;
     thePassAllFilter = nullptr;
+    logAccessor->Release();
```

Comment:
I suspect this should be
`::Release(logAccessor)
`to remove the crash.

Reply:
Done. Looked at the implementation, is it simply protecting the Release call from referencing nullptr?

Reply:
yes